deployment/spark-docker/Dockerfile

```diff
@@ -42,7 +42,9 @@ RUN wget -q -O /tmp/Miniconda2-latest-Linux-x86_64.sh \
     'ipykernel=4.3.1' 'jupyter_client=4.3.0' 'pandas=0.16*' 'matplotlib=1.4*' \
     'scipy=0.15*' 'seaborn=0.6*' 'scikit-learn=0.16*' && \
 # Install r packages
-    /opt/conda/bin/conda install --yes -c r r-essentials && \
+#    r-essentials installation hangs during installation
+#    disable it for now
+#    /opt/conda/bin/conda install --yes -c r r-essentials && \
```

Comment:
Do we now what is the cause of that? Are you able to install it on your machine?

Reply:
I don't have conda. I've tried to install that again in sessionmanager. It got stuck on solving environment.